functionary/prompt_template/__init__.py

```diff
@@ -37,16 +43,26 @@ def get_prompt_template_from_tokenizer(tokenizer: Any) -> PromptTemplate:
     p1 = PromptTemplateV1.get_prompt_template()
     p2 = PromptTemplateV2.get_prompt_template()
     p3 = Llama3Template.get_prompt_template()
+    p6 = Llama3TemplateV3.get_prompt_template()
```

Comment:
Should be p4?

Reply:
yes, p6 because I removed p4, p5 for 2 Qwen

Reply:
Are you keep Qwen stuff in a separate branch? If so, then we can keep it as p6.